tests/010-test-use-devs-to-create-docker-root-volume.sh

```diff
@@ -0,0 +1,64 @@
+source $SRCDIR/libtest.sh
+
+# Test DOCKER_ROOT_VOLUME= directive. Returns 0 on success and 1 on failure.
+test_docker_root_volume() {
+  local devs=$TEST_DEVS
+  local test_status=1
+  local testname=`basename "$0"`
+  local vg_name="dss-test-foo"
+  local docker_root_lv_name="docker-root-lv"
+
+  # Error out if any pre-existing volume group vg named dss-test-foo
+  if vg_exists "$vg_name"; then
+    echo "ERROR: $testname: Volume group $vg_name already exists." >> $LOGS
+    return $test_status
+  fi
+
+  # Create config file
+  cat << EOF > /etc/sysconfig/docker-storage-setup
+DEVS="$devs"
+VG=$vg_name
+DOCKER_ROOT_VOLUME=yes
+DOCKER_ROOT_VOLUME_SIZE=40%FREE
```

Comment:
Initially I did not specify `DOCKER_ROOT_VOLUME_SIZE=40%FREE` in the test. I was expecting docker-storage-setup to pick up the default value from `/usr/lib/docker-storage-setup/docker-storage-seutp`. However the CI started failing. I believe it was because CI version of that file did not have these new storage options `DOCKER_ROOT_VOLUME` and `DOCKER_ROOT_VOLUME_SIZE`.